frontend/src/components/charts/AreaChart/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { AreaChart as RechartsArea, Area } from "recharts";
+
+import "./styles.scss";
+
+type Props = {
+  data: {
+    name: string;
+    value: number;
+  }[];
+  width: number;
+  height: number;
+};
+
+export const AreaChart = ({ data, width, height }: Props) => {
+  return (
+    <div className="AreaChart">
+      <RechartsArea width={width} height={height} data={data}>
+        <defs>
+          <linearGradient id="colorGradient" x1="0" y1="0" x2="0" y2="1">
+            <stop offset="80%" stopColor="var(--pal-graph-gradient-primary)" />
+            <stop
+              offset="100%"
+              stopColor="var(--pal-graph-gradient-secondary)"
```

Comment:
What do you mean exactly? Should I put the color directly here instead of using the CSS variables? Or do you mean to have the variables, but control the opacity "internally" in the component using `rgba`, or something like that?

Reply:
Sorry, I should have added an example to make it clear. For transparent colors we use `rgba(var(--pal-color), [opacity])` to keep the color palette simple. Does that make sense?

Reply:
Oh, got it now. Yeah, it makes sense. I'll update this.

Reply:
I can't use the `rgba` as you said because de Sass does not do anything on this part of the code. So, I just removed the alpha channel from the HEX and added a `stopOpacity` attribute to the SVG, and got the same result.